### ISSUE TITLE
Duplicate "See further guidelines" links in CMS

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#10158]
## Description
On the edits screens in the CMS, some help text has links that for further guidelines to assist users.  Each of these links have the same link text - "See further guidelines" - and this can cause duplicate links on the same page when multiple are used.  Ideally each link should have a distinct, unique text to indicate to the user where the link would direct them - "See further guidelines about capitalization"

## Screenshot
![image](https://user-images.githubusercontent.com/106678594/184155508-d773831d-19d0-4511-bb3f-c723507d8870.png)

## Accessibility Standard

WCAG version 2.0 AAA, [Criterion 2.4.9](https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-link-only.html)

## Acceptance Criteria
- [ ] UX/IA reviewing to determine content updates for links
- [ ] Technical review
- [ ] Change management consulted
- [ ] Implementation ticket created

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide Crew`
- [ ] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public Websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`
